### PR TITLE
Improvements to user footer

### PR DESCRIPTION
**Pull request by BomberFish**
- Use `gear` instead of `gearshape` for settings icon
- Limit custom status to one line like the official client

### CODE REVIEW

**Comment by Sjmarf** [comment]
Icon comparison for convenience:

`gear`:

gear@4x

`gearshape.fill`:

gearshape fill@4x

**Comment by BomberFish** [comment]
I was (finally!) able to get Xcode builds working (updating to 14.3 final worked), so here's the user footer with my changes:

Screenshot taken on 2023-05-15 at 13 55 39

I was thinking of making the status a scrolling "marquee" text, but I felt that would be unnecessary as you could already see that in the user poppet.

**Comment by cryptoAlgorithm** [comment]
Yup, I actually decided on the filled gear as I felt that matched Discord's style better. However it's clear we are after a more native look that flows with system apps, and as such I'll probably be merging this PR.

File: Swiftcord.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -1401,13 +1401,13 @@
 				ASSETCATALOG_COMPILER_GLOBAL_ACCENT_COLOR_NAME = AccentColor;
 				CODE_SIGN_ENTITLEMENTS = Swiftcord/Swiftcord.entitlements;
 				CODE_SIGN_IDENTITY = "Apple Development";
-				"CODE_SIGN_IDENTITY[sdk=macosx*]" = "Apple Development";
+				"CODE_SIGN_IDENTITY[sdk=macosx*]" = "-";
```
Comment (cryptoAlgorithm) [inline Swiftcord.xcodeproj/project.pbxproj:1404]:
```suggestion
				"CODE_SIGN_IDENTITY[sdk=macosx*]" = "Apple Development";
```

File: Swiftcord.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -1401,13 +1401,13 @@
...
 				CODE_SIGN_STYLE = Automatic;
 				COMBINE_HIDPI_IMAGES = YES;
 				CURRENT_PROJECT_VERSION = 16;
 				DEAD_CODE_STRIPPING = YES;
 				DEVELOPMENT_ASSET_PATHS = "\"Swiftcord/Preview Content\"";
-				DEVELOPMENT_TEAM = Q382QLKDG3;
+				DEVELOPMENT_TEAM = "";
```
Comment (cryptoAlgorithm) [inline Swiftcord.xcodeproj/project.pbxproj:1410]:
```suggestion
				DEVELOPMENT_TEAM = Q382QLKDG3;
```

File: Swiftcord.xcodeproj/project.pbxproj [hunk 3]
```diff
@@ -1659,13 +1659,13 @@
 				ASSETCATALOG_COMPILER_GLOBAL_ACCENT_COLOR_NAME = AccentColor;
 				CODE_SIGN_ENTITLEMENTS = Swiftcord/Swiftcord.entitlements;
 				CODE_SIGN_IDENTITY = "Apple Development";
-				"CODE_SIGN_IDENTITY[sdk=macosx*]" = "Apple Development";
+				"CODE_SIGN_IDENTITY[sdk=macosx*]" = "-";
```
Comment (cryptoAlgorithm) [inline Swiftcord.xcodeproj/project.pbxproj:1662]:
```suggestion
				"CODE_SIGN_IDENTITY[sdk=macosx*]" = "Apple Development";
```

File: Swiftcord.xcodeproj/project.pbxproj [hunk 4]
```diff
@@ -1659,13 +1659,13 @@
...
 				CODE_SIGN_STYLE = Automatic;
 				COMBINE_HIDPI_IMAGES = YES;
 				CURRENT_PROJECT_VERSION = 16;
 				DEAD_CODE_STRIPPING = YES;
 				DEVELOPMENT_ASSET_PATHS = "\"Swiftcord/Preview Content\"";
-				DEVELOPMENT_TEAM = Q382QLKDG3;
+				DEVELOPMENT_TEAM = "";
```
Comment (cryptoAlgorithm) [inline Swiftcord.xcodeproj/project.pbxproj:1668]:
```suggestion
				DEVELOPMENT_TEAM = Q382QLKDG3;
```

File: Swiftcord.xcodeproj/project.pbxproj [hunk 5]
```diff
@@ -1693,13 +1693,13 @@
 				ASSETCATALOG_COMPILER_GLOBAL_ACCENT_COLOR_NAME = AccentColor;
 				CODE_SIGN_ENTITLEMENTS = Swiftcord/Swiftcord.entitlements;
 				CODE_SIGN_IDENTITY = "Apple Development";
-				"CODE_SIGN_IDENTITY[sdk=macosx*]" = "Apple Development";
+				"CODE_SIGN_IDENTITY[sdk=macosx*]" = "-";
```
Comment (cryptoAlgorithm) [inline Swiftcord.xcodeproj/project.pbxproj:1696]:
```suggestion
				"CODE_SIGN_IDENTITY[sdk=macosx*]" = "Apple Development";
```

File: Swiftcord.xcodeproj/project.pbxproj [hunk 6]
```diff
@@ -1693,13 +1693,13 @@
...
 				CODE_SIGN_STYLE = Automatic;
 				COMBINE_HIDPI_IMAGES = YES;
 				CURRENT_PROJECT_VERSION = 16;
 				DEAD_CODE_STRIPPING = YES;
 				DEVELOPMENT_ASSET_PATHS = "\"Swiftcord/Preview Content\"";
-				DEVELOPMENT_TEAM = Q382QLKDG3;
+				DEVELOPMENT_TEAM = "";
```
Comment (cryptoAlgorithm) [inline Swiftcord.xcodeproj/project.pbxproj:1702]:
```suggestion
				DEVELOPMENT_TEAM = Q382QLKDG3;
```

**Review by cryptoAlgorithm — CHANGES_REQUESTED** [review]
Please do not commit modifications to code signing settings (including development team) to prevent breaking CI builds and for consistency.

**Review by cryptoAlgorithm — CHANGES_REQUESTED** [review]
Please do not commit modifications to code signing settings (including development team) to prevent breaking CI builds and for consistency.

**Pull request merged**